### PR TITLE
Document Content Validator #335

### PR DESCRIPTION
**Pull request by rymsha**

### CODE REVIEW

File: docs/cms/validators.adoc [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        .generalError( ValidationErrorCode.from( ApplicationKey.from( "my.app" ), "custom.invalid" ) )
+        .i18n( "my.app.custom.invalid" )
+        .message( "{0} is not a dog" )
+        .args( "cat" )
+        .build();
+----
+
+Validation error must be identified with ValidationErrorCode, that consists of `ApplicationKey` and code string. `ApplicationKey` can be used to identify Localization bundle for the error message.
+
+`i18n` filed defiles i18n key and can be used to identify a message key in Localization bundle.
```
Comment (alansemenov) [inline docs/cms/validators.adoc:68]: What is "_filed defiles_"? "_file defines_"?

File: docs/cms/validators.adoc [hunk 2]
```diff
@@ -0,0 +1,83 @@
+= Content Validators
+:toc: right
+:imagesdir: ../images
+
+image:xp-780.svg[XP 7.8.0,opts=inline]
+
+== Introduction
+
+XP by default validates nearly created and modified content:
```
Comment (alansemenov) [inline docs/cms/validators.adoc:9]: What is "_nearly created_"? "_nearly **all** created_"?

File: docs/cms/validators.adoc [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        .message( "{0} is not a dog" )
+        .args( "cat" )
+        .build();
+----
+
+Validation error must be identified with ValidationErrorCode, that consists of `ApplicationKey` and code string. `ApplicationKey` can be used to identify Localization bundle for the error message.
+
+`i18n` filed defiles i18n key and can be used to identify a message key in Localization bundle.
+
+NOTE: Applications like ContentStudio use `ValidationErrorCode` as a fallback to _guess_ i18n key, for instance if `ValidationErrorCode` is `my.app:custom.invalid` fallback i18k for it is `my.app.custom.invalid`.
```
Comment (alansemenov) [inline docs/cms/validators.adoc:70]: > if `ValidationErrorCode` is `my.app:custom.invalid` fallback i18k for it is

if `ValidationErrorCode` is `my.app:custom.invalid` then fallback i18n key for it is

File: docs/release.adoc [hunk 4]
```diff
@@ -9,6 +9,8 @@ include::variables.adoc[]
 
 Library for creating and importing <<api/lib-export#_export_library, node-exports>> is now available.
 
+Java <<cms/validators#_content_validators, Content Validators API>> in now available.
```
Comment (alansemenov) [inline docs/release.adoc:12]: _is now available_

**Review by alansemenov — CHANGES_REQUESTED** [review]

File: docs/cms/validators.adoc (hunk 2 above)
Comment (rymsha) [reply docs/cms/validators.adoc:9]:
```suggestion
XP by default validates newly created and modified content:
```

File: docs/cms/validators.adoc (hunk 1 above)
Comment (rymsha) [reply docs/cms/validators.adoc:68]:
```suggestion
`i18n` field defiles i18n key and can be used to identify a message key in Localization bundle.
```

File: docs/cms/validators.adoc (hunk 3 above)
Comment (rymsha) [reply docs/cms/validators.adoc:70]:
```suggestion
NOTE: Applications like ContentStudio use `ValidationErrorCode` as a fallback to _guess_ i18n key, for instance if `ValidationErrorCode` is `my.app:custom.invalid` fallback i18n for it is `my.app.custom.invalid`.
```

**Pull request merged**